spark/v3.5/spark-runtime/NOTICE

```diff
@@ -500,9 +363,312 @@ file:
 This binary artifact includes Project Nessie with the following in its NOTICE
 file:
 
-| Dremio
-| Copyright 2015-2017 Dremio Corporation
-|
-| This product includes software developed at
-| The Apache Software Foundation (http://www.apache.org/).
-
+| Nessie
+| Copyright 2015-2025 Dremio Corporation
+| 
+| ---------------------------------------
+| This project includes code from Apache Iceberg, with the following in its NOTICE file:
```

Comment:
This can be trimmed because it is from our own NOTICE. We are responsible for all the content about what Iceberg ships, so there is no need to duplicate.

Reply:
I trimmed the Nessie `NOTICE` by removing Iceberg and Netty (as it's already mentioned in the Iceberg Spark runtime `NOTICE`). I keep Polaris mention as it's not duplicate (ok, it's about build/gradle resources in Polaris, but for consistency, better to keep).